documentation/modules/ROOT/pages/reactive.adoc

```diff
@@ -2,7 +2,9 @@
 
 Quarkus provides a novel reactive API called Mutiny, with the goal of easing the development of highly scalable, resilient, and asynchronous systems.
 
-In this chapter we're going to see some examples of how Mutiny changes the design of our Quarkus applications. We're going to develop an asynchronous REST client that connects to an online beer database (https://punkapi.com/documentation/v2) to retrieve beer information. This API does not return all beers at once, so we'll need to navigate through the pages to fetch all the information. Then we're going to filter all the beers with an ABV greater than 15.0 and return these values in a Reactive REST endpoint.
+In this chapter we're going to see some examples of how Mutiny changes the design of our Quarkus applications. 
+to https://www.fruityvice.com/ to retrieve fruit information based on their carbohydrates. 
```

Comment:
Some words are missing here I believe

Reply:
Yes, initially I tried to reuse the fruityvice API but it did not serve the purpose well.